test/test_helper.rb

```diff
@@ -2,4 +2,10 @@
 require "r2d2"
 
 require "minitest/autorun"
-require "pry-byebug"
+
+begin
+  require "pry-byebug"
+rescue LoadError
+  # Ignore, byebug is not installed for older ruby versions
+end
+
```

Comment:
I would love to see dev/debug-specific tooling like pry removed all together. Any thoughts on that?

Reply:
I'm ok with this, because I'm used to working with older ruby versions where we can't use these tools anyway.